src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -533,6 +521,16 @@ public void SolutionBindingUpdated_WhenSetBoundProject_EventsRaisedInExpectedOrd
             }
         }
 
+        private static ActiveSolutionBoundTracker CreateTestSubject(IHost host,
```

Comment:
Helper method to avoid having to change every test to support the new parameter.